menas/src/test/scala/za/co/absa/enceladus/menas/integration/controllers/BaseRestApiTest.scala

```diff
@@ -151,20 +166,30 @@ abstract class BaseRestApiTest extends BaseRepositoryTest {
     Future { send(method, urlPath, headers, bodyOpt) }
   }
 
-  def send[B, T](method: HttpMethod, urlPath: String, headers: HttpHeaders = HttpHeaders.EMPTY,
-                 bodyOpt: Option[B] = None)(implicit ct: ClassTag[T]): ResponseEntity[T] = {
+  private def sendBase[T, B](method: HttpMethod, urlPath: String, headers: HttpHeaders, bodyOpt: Option[B], ct: ClassTag[T]) = {
```

Comment:
Just curious, nothing bad with it: why you opted to switch to explicit `ct` in this sub-function?

Reply:
Automatic Intellij Method extract did this